Update information about Viessmann developer dashboard in ViCare integration (#31059)
home-assistant/home-assistant.github.io: source/_integrations/vicare.markdown

{% include integrations/config_flow.md %}

Use your ViCare app login credentials for *username* and *password*.

The required *client ID* can be obtained from the [Viessmann Developer Portal](https://app.developer.viessmann.com/):

> 1. Login with **your existing ViCare app username/password**.

Reviewer: "Log in" or "Log-in" would be more correct here. (verb instead of noun)
```suggestion
1. Log in with **your existing ViCare app username/password**.
```

> 1. Log in with **your existing ViCare app username/password**.

Author: ```suggestion
1. Log in with **your existing ViCare app username and password**.
```

> 2. On the developer dashboard click *add* in the *clients* section.

Reviewer: minor, but I'd probably add a comma here:
```suggestion
2. On the developer dashboard, click *add* in the *clients* section.
```

---

Also, looking at the [Google Assistant docs](https://www.home-assistant.io/integrations/google_assistant/#google-cloud-platform-configuration) as an example, bold text is used for the "buttons" (rather than italic). They're also capitalized. So, this seems like it would be a better fit:

```suggestion
2. On the developer dashboard, click **Add** in the **Clients** section.
```

> 2. On the developer dashboard, click **Add** in the **Clients** section.

Reviewer 2: ```suggestion
2. On the developer dashboard, select **Add** in the **Clients** section.
```
tiny tweak. We usually don't use "click", as it is more mouse-specific. See [MS style guide](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/click)

> 3. Create a new client using following data:

Reviewer: I think this is more common:
```suggestion
3. Create a new client using the following data:
```

   - Name: `HomeAssistant`
   - Google reCAPTCHA: Disabled
   - Redirect URIs: `vicare://oauth-callback/everest`

> 4. Find the *client ID* in the *clients* section on the developer dashboard.

Reviewer: Same would go here then (with bold and capitalized text for "buttons"):
```suggestion
4. Find the **Client ID** in the **Clients** section on the developer dashboard.
```
